source/direct_debit/index.html.md.erb

```diff
@@ -5,285 +5,259 @@ weight: 80
+GoCardless submits a request to BACS to set up a mandate, and
+sends a mandate ID to GOV.UK Pay. At this point, you can send a payment
+request.
 
-These steps happen in order.
+### 6. GOV.UK Pay returns the user to your service using the `return_url`
 
-#### 1. GOV.UK Pay creates the agreement
+### 7. Call the API to check the status of the mandate
 
-API call:  `POST /v1/agreements/`
+API call: `GET /v1/directdebit/mandates/<mandateid>`
 
-Your service must specify:
+The API response includes a `state` object. This includes the following parameters:
 
-* a `return_url` where GOV.UK Pay will direct the user when they have finished setting up the agreement
+- [`status`](#mandate-statuses)
+- [`details`](https://developer.gocardless.com/api-reference/#events-mandate-causes), which explains the reason for the `status`
 
-* the `agreement_type` (`on demand`)
+You should use these `status` and `details` parameters to display an appropriate final page to your user.
 
-* a `reference` (optional) - your reference for this agreement
+The `status` will be `pending` until the user's bank confirms the mandate. The mandate confirmation process takes 2 working days.
 
-The response contains a `next_url`.
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response.
 
-#### 2. Your service directs the user to the `next_url`
+You can also check the status of a mandate by signing into your GoCardless account through the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
 
-#### 3. The user provides their account details
 
-For example, account number and sort code for Direct Debit.
+## Collecting payments against a mandate
 
-GOV.UK Pay makes an API call to our Direct Debit supplier to verify account
-details.
+API call: `POST /v1/directdebit/payments`
 
-#### 4. GOV.UK Pay presents the user with a confirmation screen
+Required arguments:
 
-The screen presents the user’s account details. The user grants their consent
-by selecting __Confirm__.  GOV.UK Pay sends an email to the user to confirm
-that their mandate has been set up. If the mandate subsequently fails, the
-GOV.UK Pay team will send an email to update the user that their mandate could
-not be set up.
+```
+{
+   "amount"
+   "reference"
+   "mandate_id"
+}
+```
 
-#### 5. GOV.UK Pay securely passes the user's details on to the Direct Debit provider
+Optional argument:
 
-The Direct Debit provider submits a request to BACS to set up a mandate, and
-sends a mandate ID to GOV.UK Pay. At this point, services can send a payment
-request.
+```
+{
+   "description"
+}
+```
 
-#### 6. GOV.UK Pay returns the user to your service using the `return_url`
+Once a mandate has a `status` of `pending`, you can make an API call to collect a payment against the mandate.
```

Comment:
Do we want to explicitly state here that any attempt to collect a payment against a mandate can fail?

Reply:
I'm not sure we need it here - the '## Collecting payments against a mandate' section talks about `status`, `detail` and `finished` states (and links to the various status/detail codes), so it should be clear that a payment could fail. Is that ok?

Reply:
Given we’re focusing on the happy path, I think that’s fine.